cmd/clusterlink/cmd/create/create.go

```diff
@@ -20,11 +20,13 @@ import (
 // NewCmdCreate returns a cobra.Command to run the create command.
 func NewCmdCreate() *cobra.Command {
 	cmds := &cobra.Command{
-		Use: "create",
+		Use:   "create",
```

Comment:
nit: I think `Use` should be more detailed (see [description in Cobra docs](https://github.com/spf13/cobra/blob/bd914e58d69d65e494b45bdb40e90ca816b92fcc/command.go#L52))